src/data/faq.json

```diff
@@ -446,6 +446,18 @@
                             "<ol><li>Go to your device settings</li><li>Select 'Security & Location' -> 'Location' -> 'App level permissions'</li></ol>",
                             "Here you can deny other apps permission to use your location if you don't think they need it. Since Corona-Warn-App doesn't need this permission, it doesn't appear in the list."
                         ]
+                    },
+                    {
+                        "title": "[Google/Android] A card on my Start Screen is notifying me that my device is not or only partially compatible. What does this mean?",
+                        "anchor": "incompatibility_warning",
+                        "active": true,
+                        "textblock": [
+                            "If you see such a notice, your device is not able to send any random IDs via Bluetooth. This can be caused by a restriction due to your combination of hardware and software. Because of it, the Corona-Warn-App cannot use Bluetooth Low Energy in its so-called “periphial mode”.",

```

Comment:
https://source.android.com/devices/bluetooth/ble refers to "BLE peripheral mode". The spelling "“periphial" doesn't look right.

Reply:
Sorry, I made this mistake again :x